src/olympia/amo/tests/__init__.py

```diff
@@ -1013,6 +964,56 @@ def setUpClass(cls):
         # right before each test.
         stop_es_mocks()
         cls.es = amo_search.get_es()
+        # Make sure ES cluster is in a good state, resetting the index if
```

Comment:
what does refactoring this code from `setup_es_data` (that was called from `setUpTestData` instead) change?

Reply:
`setUpTestData` was not called in `TestIndexCommand` because that's a `TransactionTestCase`, and `setUpTestData` is a `TestCase` thing. Moving it to `setUpClass` ensures it is called, which in turns is useful to ensure the tests start from a clean state.

The problem wasn't visible in CI, but locally if you ran the tests multiple times and had some test failures while developing, you could end up in a state where the `TestIndexCommand` would repeatedly fail because an existing index was set up already. Before that change, the only way to get out of that was to execute _another_ test with elasticsearch enabled that wasn't a `TransactionTestCase`, or manually clean up your ES indexes...